README.md

```diff
@@ -30,11 +30,11 @@
 `teal.code` is an R library providing tools to store code and an execution environment associated with it. The features
 include:
 
-* an object `qenv` for storing code and an execution environment which integrates well with `shiny reactives` for use in `shiny` applications whose outputs require reproducibility (i.e. the code used to generate them)
+* an object `qenv` for storing code and an execution environment that integrates well with `shiny reactives` for use in `shiny` applications whose outputs require reproducibility (i.e. the code used to generate them)
 * ability to chain and join `qenv` objects together to provide fine-grained control over executed code
 * automatic error and warning handling for executed code
 
-`teal.code` also ships a [`shiny`](https://shiny.posit.co/) module that helps inspect the stored code as well as messages, warnings and error messages resulting from evaluation via `shiny` web application.
+`teal.code` also ships a [`shiny`](https://shiny.posit.co/) module that helps inspect the stored code as well as messages, warnings, and error messages resulting from evaluation via `shiny` web application.
```

Comment:
Some part of this is valid and information was already conveyed for `qenv`, And this text was [added](https://github.com/insightsengineering/teal.code/commit/71c737a85d2895a1e15c3fa356b23a144d49a998#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5) when we had `chunks` before` qenv`. I am not 100% sure it's completely valid. I think it can be removed . WDYT?

Reply:
I believe this section is outdated. I noticed some outdated elements in `teal.code` that should be removed in a separate issue.
#190

Reply:
Never mind, false alarm. Just remove this section.

Reply:
[31322dc](https://github.com/insightsengineering/teal.code/pull/189/commits/31322dce9ba53c53d077cd2586e36ba4861e12f7)